aa-sdk/core/src/account/smartContractAccount.ts

```diff
@@ -146,6 +146,18 @@ export const parseFactoryAddressFromAccountInitCode = (
   return [factoryAddress, factoryCalldata];
 };
 
+/**
+ * Retrieves the account address. Uses a provided `accountAddress` if available; otherwise, it computes the address using the entry point contract and the initial code.
+ *
+ * @example TODO: IMPLEMENT ME
```

Comment:
TODO: make this specific message a lint error so that it gets caught before being merged